src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -2508,6 +2512,12 @@ protected override IGroupedAnalyzerActions CreateGroupedActions(DiagnosticAnalyz
             return success;
         }
 
+        private SemanticModel GetOrCreateSemanticModel(SyntaxTree tree, Compilation compilation)
```

Comment:
>GetOrCreateSemanticModel [](http://example.com/codeflow?start=30&length=24)

Is this a clone of the base class method? If so, could we make the base method `protected` instead of redeclaring?